wiring/inc/spark_wiring_usartserial.h

```diff
@@ -72,6 +72,10 @@ class USARTSerial : public Stream
   hal_usart_interface_t interface() const {
     return _serial;
   }
+
+  operator hal_usart_interface_t() const {
```

Comment:
Let's not add implicit conversion operators. There is already `interface()` function above as well.

Reply:
I was thinking that we have the same operators for network class, to be consistent and convenient for user application to just pass `Serial1` as the `.usart()` parameter.

Reply:
In that case it's better for the sleep configuration object to take wiring class type instead and perform necessary conversions.